ios/jailed/patchapp.sh

```diff
@@ -0,0 +1,377 @@
+#!/bin/bash
```

Comment:
Honestly, I barely know makefile (I've been learning by going through Theos' code), so I don't think I'll be able to convert a bash script into a makefile. I could add a makefile that launches the script though.